src/main/java/com/bamdoliro/gati/domain/board/presentation/dto/response/BoardDetailDto.java

```diff
@@ -0,0 +1,19 @@
+package com.bamdoliro.gati.domain.board.presentation.dto.response;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class BoardDetailDto {
+
+    private String writer;
+    private String title;
+    private String content;
+
+    @Builder
+    public BoardDetailDto(String writer, String title, String content) {
+        this.writer = writer;
+        this.title = title;
+        this.content = content;
+    }
+}
```

Comment:
이렇게 모든 멤버변수에 `@Builder` annnotation을 사용하는 경우라면
그냥 class위에 달아주면 간결한 코드가 됩니다.

```suggestion
@Getter
@Builder
public class BoardDetailDto {

    private String writer;
    private String title;
    private String content;
}
```

Reply:
그렇네요.. 감사합니다